docs/ventoy.md

```diff
@@ -1,12 +1,12 @@
 ---
-title: Installing Ventoy in chromeOS
+title: Installing Ventoy in ChromeOS
```

Comment:
prettt sure we are supposed to only use "chromeOS"

Reply:
Sure, I'll change the other instance of it too